### DISCUSSION TITLE
error in the launching process

### DISCUSSION THREAD

**Discussion by Theophilesncfscrap** [omkarcloud/google-maps-scraper#167] [Q&A] [answered]
Hello, is anyone can help me with this issue. Last week I was able to use it properly and i have somes scraps that i want to recover from my previous sessions. 

Thanks in advance 

<img width="671" alt="error okmar cloud" src="https://github.com/omkarcloud/google-maps-scraper/assets/168532780/619c6796-93a2-4772-a864-81a118c13d95">

**Comment by Chetan11-dev** [comment, marked as answer]
Run python -m pip install botasaurus  botasaurus-requests --upgrade
